src/index.html

```diff
@@ -20,8 +20,312 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#home"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo-logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul>
+          <li class="nav__item">
+            <a
+              class="is-active"
+              href="#apple"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops-computers"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="card-container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="'iMac with Retina 5K display"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <div class="card__code">
+          <p class="card__code-text">Product code: 195434</p>
+        </div>
+        <div class="card__stars-reviews-wrapper">
+          <div class="card__stars card__stars--4">
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+          </div>
+          <div class="card__reviews">
+            <p class="card__reviews-text">Reviews: 5</p>
+          </div>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-number">$2,199</p>
+        </div>
+        <a
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="'iMac with Retina 5K display"
```

Comment:
There is a typo in the alt attribute value. The single quote before 'iMac' should be removed.